src/main/java/frc/robot/Constants.java

```diff
@@ -59,4 +65,63 @@ public static class Elevator {
         public static final Dimensionless GEAR_RATIO = Rotations.of(0).div(Rotations.of(1)); // output over input
         public static final Angle MOTOR_MAX_HEIGHT = (Angle) MAX_HEIGHT.div(ELEVATOR_EXTENSION_PER_MOTOR_ANGLE).div(GEAR_RATIO);
     }
+
+    public static final class Pivot {
+        // TODO: set all of these constants, CAN IDs, and ports
+        public static final int PIVOT_MOTOR_ID = 10;
+        public static final int ROLLER_MOTOR_ID = 11;
+        
+        public static final int BEAM_BREAK_PORT = 0; 
+        
+        public static final double PIVOT_GEAR_RATIO = 0.0;
+        
+        // motion and position control TODO: sam pls check im using these correctly        
+        public static final Measure<AngleUnit> MIN_ANGLE = Degrees.of(0);
+        public static final Measure<AngleUnit> MAX_ANGLE = Degrees.of(0);
+        public static final Measure<AngleUnit> ANGLE_TOLERANCE = Degrees.of(0);
+        
+        // feedforward and pid constants
+        public static final double kG = 0.0;
+        public static final double kS = 0.0;
+        public static final double kV = 0.0;
+        public static final double kA = 0.0;
+        
+        public static final double kP = 0.0;
+        public static final double kI = 0.0;
+        public static final double kD = 0.0;
```

Comment:
we should probably just modify the PIDConfig class to be able to store G, S, V, and A gains as well or something